internal/webhook/deployment_webhook.go

```diff
@@ -59,6 +59,9 @@ func (v *DeploymentValidator) ValidateCreate(ctx context.Context, obj runtime.Ob
 	if !ok {
 		return nil, apierrors.NewBadRequest(fmt.Sprintf("expected Deployment but got a %T", obj))
 	}
+	if deployment.Spec.Template == "" {
```

Comment:
Template is already set as "required" in the openapi schema, so this one is unnecessary

Reply:
You're right. However, I found that it's no longer working when the admission webhooks are enabled. I'll double-check.